tests/introspection/Mac/MacApiProtocolTest.cs

```diff
@@ -171,6 +171,13 @@ protected override bool Skip (Type type, string protocolName)
 					return true; 
 				}
 				break;
+			case "NSCandidateListTouchBarItemDelegate":
```

Comment:
Not sure this is kosher but...

***** ApiProtocolTest.GeneralCase
[FAIL] Type AppKit.NSTextView (native: NSTextView) does not conform NSCandidateListTouchBarItemDelegate
[FAIL] Type AppKit.NSTextView (native: NSTextView) does not conform NSTouchBarDelegate

@interface NSTextView (NSTextView_TouchBar) <NSCandidateListTouchBarItemDelegate, NSTouchBarDelegate>